client/src/components/App/AppToolbar/useAppToolbar.tsx

```diff
@@ -23,15 +24,14 @@ const useAppToolbar = () :  useTopToolbarOutcome => {
     const user = useSelector<StoreStateType, User>(state => state.user);
     const classes = useStyles();
     
-    const [isActive, setIsActive] = React.useState<boolean | null>(null);
     const [countyDisplayName, setCountyDisplayName] = React.useState<string>('');
 
     React.useEffect(() => {
         if (user.investigationGroup !== -1) {
             getCountyByUser();
             getUserActivityStatus();
         }
-    }, [user]);
+    }, [user.investigationGroup]);
```

Comment:
why on investigationGroup change?

Reply:
because userActivityStatus changes the user redux state, and it got into infinite loop

Reply:
sure, but why investigationGroup?

Reply:
because this is the field in the condition so only when this field changes I need to reconsider fetching the data